<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsGui</class>
 <widget class="QDialog" name="settingsGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>settingsView</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="serverTab">
    <attribute name="title">
     <string>Server</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>531</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string> Main Server Directory</string>
     </property>
     <widget class="QLineEdit" name="directoryInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Select the folder you wish to set up as the main FTP directory.</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="directorySelector">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="minimizeToTray">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>351</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimize the server to system tray on close.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="usersTab">
    <attribute name="title">
     <string>Users</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>421</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>User Manager</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="registerUserButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Create User</string>
      </property>
     </widget>
     <widget class="QPushButton" name="directoryPermittedSelector">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>If no specific directory is selected, the user will have access to all of the servers directories.</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="directoryPermittedInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>If no specific directory is selected, the user will have access to all of the servers directories.</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>If left empty directory defaults to the Main FTP directory.</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>If no specific directory is selected, the user will have access to all of the servers directories.</string>
      </property>
      <property name="text">
       <string>Directory Permitted</string>
      </property>
     </widget>
     <widget class="QLabel" name="userValidationLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>185</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color:Red; </string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>161</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>Registered Users</string>
     </property>
     <widget class="QListWidget" name="userListWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>121</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>480</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>480</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
